src/Misc/layoutbin/installdependencies.sh

```diff
@@ -0,0 +1,158 @@
+            if [ $? -ne 0 ]
+            then
+                echo "'apt' failed with exit code '$?'"
+                print_errormessage
+                exit 1
+            fi
+        else
+            command -v apt-get
+            if [ $? -eq 0 ]
+            then
+                apt-get update && apt-get install -y libunwind8-dev gettext libicu-dev liblttng-ust-dev libcurl4-openssl-dev libssl-dev uuid-dev unzip
+                if [ $? -ne 0 ]
+                then
+                    echo "'apt-get' failed with exit code '$?'"
+                    print_errormessage
+                    exit 1
+                fi
+            else
+                echo "Can not find 'apt' or 'apt-get'"
+                print_errormessage
+                exit 1
+            fi
+        fi
+    elif [ -e /etc/redhat-release ]
+    then
+        echo "The current OS is Fedora based"
+        echo "--------Redhat Version--------"
+        cat /etc/redhat-release
+        echo "------------------------------"
+
+        # use dnf on fedora
+        # use yum on centos and redhat
+        if [ -e /etc/fedora-release ]
+        then
+            command -v dnf
+            if [ $? -eq 0 ]
+            then
+                grep -i 'fedora release 26' /etc/fedora-release
+                if [ $? -eq 0 ]
+                then
+                    echo "Use compat-openssl10-devel instead of openssl-devel for Fedora 26 (dotnet core requires openssl 1.0.x)"
+                    
+                    # epel-release doesn't exists in fedora's package repository, so we can't use dnf to install it
+                    rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && dnf install -y deltarpm unzip libunwind gettext libcurl-devel compat-openssl10 zlib libicu-devel
+                    if [ $? -ne 0 ]
+                    then
+                        echo "'dnf' failed with exit code '$?'"
+                        print_errormessage
+                        exit 1
+                    fi
+                else
+                    # epel-release doesn't exists in fedora's package repository, so we can't use dnf to install it
+                    rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && dnf install -y deltarpm unzip libunwind gettext libcurl-devel openssl-devel zlib libicu-devel
+                    if [ $? -ne 0 ]
+                    then
+                        echo "'dnf' failed with exit code '$?'"
+                        print_errormessage
+                        exit 1
+                    fi
+                fi                
+            else
+                echo "Can not find 'dnf'"
+                print_errormessage
+                exit 1
+            fi
+        else
+            command -v yum
+            if [ $? -eq 0 ]
+            then
+                yum install -y deltarpm epel-release unzip libunwind gettext libcurl-devel openssl-devel zlib libicu-devel
```

Comment:
libunwind

Reply:
lttng-ust doesn't exist in standard YUM repo

Reply:
libcurl

Reply:
openssl-libs

Reply:
libuuid

Reply:
krb5-libs

Reply:
zlib

Reply:
libicu

Reply:
yum install wget && wget -P /etc/yum.repos.d/ https://packages.efficios.com/repo.files/EfficiOS-RHEL7-x86-64.repo &&   rpmkeys --import https://packages.efficios.com/rhel/repo.key && yum updateinfo && yum -y install lttng-ust